src/console/controllers/DbController.php

```diff
@@ -214,7 +214,7 @@ public function actionBackup(?string $path = null): int
             $db->backupTo($path);
             if ($this->zip) {
                 $zipPath = FileHelper::zip($path);
-                unlink($path);
+                FileHelper::unlink($path);
```

Comment:
Using helper, as theres a chance `$path` will be a directory if you set `backupCommandFormat=directory`